list

```diff
@@ -665,6 +678,7 @@ kkstre.am
 kl.ik.my
 klck.me
 kli.cx
+klikny.ru
```

Comment:
need proof for this one

Reply:
Original link (https://klikny.ru/yUJ) still works for me, instant 301 redirect.
Archive: https://archive.is/DJh3i